### PR TITLE
The Following is a Suggested intro to the README.

### PR DESCRIPTION
**Pull request by LuisRoyZulu06**
This is a suggested introduction to the READ_ME File for the Zambian Conference Website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+    - What the conference will be all about.
+    - Why the conference exists.
+    - What it is trying to achieve especially    where technology is concerned.
+    - Where it will be held and time.
+    - Who is eligible to attend.
+    - What happens after the conference.
+    - Contact details of the organizers.
+    - Help every developer out there especially newbies on how to develops and deploy a website on the internet.
 
+NB: SO ITS SAFE TO SAY THAT IT WILL ALSO BE A VERY BENEFICIAL PLATFORM TO LEARN FROM.
```
Comment (OlivierJM) [inline README.md]: This section can also be removed.

**Comment by OlivierJM** [comment]
hi @LuisRoyZulu06 

Check this guide on how you can efficiently use markdown https://guides.github.com/features/mastering-markdown/

File: README.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+    - What it is trying to achieve especially    where technology is concerned.
+    - Where it will be held and time.
+    - Who is eligible to attend.
+    - What happens after the conference.
+    - Contact details of the organizers.
+    - Help every developer out there especially newbies on how to develops and deploy a website on the internet.
 
+NB: SO ITS SAFE TO SAY THAT IT WILL ALSO BE A VERY BENEFICIAL PLATFORM TO LEARN FROM. 
 
+The website will be build on MySQL database being managed via PHP, with a great UI supported by JavaScript, HTLM and CSS. We hope that this open source project could be a great hands on opportunity for many developers out there especially the beginners, as we will develop and deploy a live website on the internet together as a community.
```
Comment (OlivierJM) [inline README.md]: @LuisRoyZulu06 to make it a bit more clear, we won't use PHP nor MySQL
You can replace the above paragraph with this:

**Technologies**
 
- React
- Bootstrap
- Netlify

Comment (LuisRoyZulu06) [reply README.md]: Okay thank you very for the clarification

Comment (OlivierJM) [reply README.md]: Let me know if you know clarity on anything else.

**Comment by LuisRoyZulu06** [comment]
How do I go back and edit the README file?

**Comment by OlivierJM** [comment]
You can edit as you did at first just make sure you push to this same branch.

**Comment by OlivierJM** [comment]
The file is already saved in markdown it’s just a matter of using markdown syntax in the readme, if you followed the link I posted on top, it should have given you a clear picture of how markdown works and how to use.

**Comment by LuisRoyZulu06** [comment]
Okay thank you

**Comment by OlivierJM** [comment]
@LuisRoyZulu06  check the changes and let me know if the markdown syntax makes sense for you.

**Comment by LuisRoyZulu06** [comment]
Wow this is so simplified and straight to the point. This is good.

I have always thought a README should be long and very explanatory.

Thank you so much.

**Comment by OlivierJM** [comment]
@allcontributors please add @LuisRoyZulu06  for doc

**Pull request merged**